priv/apps/itest/test/features/in_flight_exits.feature

```diff
@@ -1,6 +1,4 @@
 Feature: In Flight Exits
-  # NB: Since the in flight transaction doesn't exist in the system
```

Comment:
idk what this means

Reply:
@InoMurko I can explain this. This test was to mimic a scenario where you had an "in flight transaction". In this case, its just a transaction that's never actually submitted/mined into the network, therefore nothing is there to actually exit.